docs/cedarling/getting-started/java.md

```diff
@@ -0,0 +1,206 @@
+---
+tags:
+  - cedarling
+  - java
+  - getting-started
+---
+
+# Getting Started with Cedarling Java
+
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Installation 
+
+## Building from Source
+
+Refer following [guide](../uniffi/cedarling-kotlin.md#building-from-source) for steps to build the Java binding from source.
+
+## Using Cedarling-java Maven dependency
+
+### Prerequisites
+
+- Java Development Kit (JDK): version 11 or higher
+
+To use Cedarling Java bindings in Java Maven Project add following `repository` and `dependency` in pom.xml of the project
+
+```declarative
+    <repositories>
+        <repository>
+            <id>jans</id>
+            <name>Janssen project repository</name>
+            <url>https://maven.jans.io/maven</url>
+        </repository>
+    </repositories>
+```
+```declarative
+        <dependency>
+            <groupId>io.jans</groupId>
+            <artifactId>cedarling-java</artifactId>
+            <version>{latest-jans-stable-version}</version>
+        </dependency>
+```
+
+## Usage
+
+### Initialization
+
+We need to initialize Cedarling first.
+
+```java
+
+        import uniffi.cedarling_uniffi.*;
+
+        String bootstrapJsonStr = """
```

Comment:
Please add a comment here that bootstrap config should be loaded dynamically in production, or show a way to load from JSON or env variables.